NumTrip/Spielfeld.py

```diff
@@ -0,0 +1,58 @@
+
+feld =[[2,4,4,4,2],[1244,2,4,2,2],[2,2,4,4,2],[2,4,4,4,4],[4,4,4,4,4]]
+
+#print('hallo', Spielfeld[0][0])
+#print(f'hallo      {Spielfeld[0][0]}', )  #formatiert
+
+def spielfeld():
+    zeilennummer=1
+    print('          1      2      3      4      5')
+    for zeile in feld:
+        print('      +------+------+------+------+------+')
+        print('      |      |      |      |      |      |')
+        print(f'   {(zeilennummer)}  ', end='')
+        for zelle in zeile:
+            if zelle >= 10000:
+                print(f'| {(zelle)}', end='')
+            elif zelle >= 1000:
+                print(f'| {(zelle)} ', end='')
+            elif zelle >= 100:
+                print(f'|  {(zelle)} ', end='')
+            elif zelle >= 10:
+                print(f'|  {(zelle)}  ', end='')
+            else:
+                print(f'|   {(zelle)}  ', end='')
+        print('|')
+        zeilennummer= zeilennummer+1
+        print('      |      |      |      |      |      |')
+    print('      +------+------+------+------+------+')
+
+
+def transform_eingabe(raw):
+    raw = raw.upper()
+    raw = raw.replace(' ','').replace('-','').replace('.','').replace(',','').replace('/','').replace(';','').replace(':','')
+    x = raw[0]
+    if not x.isnumeric():
+        print('Keine Zahl...')
+        return False
```

Comment:
und, damit Sie dann die Eingabe korrekt zuweisen können ( `x, y=transform_eingabe(raw)`), müssen Sie in jedem Fall eine Liste zurückgeben:

```suggestion
        return [False, False]
```